weave_demo.ipynb

```diff
@@ -0,0 +1,1199 @@
+   "metadata": {},
+   "source": [
+    "## Creating and Uploading Baskets\n",
+    "A Basket is a representation of an atomic data product within a pantry. Below we will demonstrate how to upload a basket containing our dummy data, called WeaveDemoText, using a pantry object. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "5f2d6c18-ad13-4d76-a2f0-8f11518af887",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>uuid</th>\n",
+       "      <th>upload_time</th>\n",
+       "      <th>parent_uuids</th>\n",
+       "      <th>basket_type</th>\n",
+       "      <th>label</th>\n",
+       "      <th>weave_version</th>\n",
+       "      <th>address</th>\n",
+       "      <th>storage_type</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>acc530466eb811eeb8b60242ac1f000a</td>\n",
+       "      <td>2023-10-19 19:49:54.672040+00:00</td>\n",
+       "      <td>[]</td>\n",
+       "      <td>test-1</td>\n",
+       "      <td></td>\n",
+       "      <td>1.1.0</td>\n",
+       "      <td>weave-demo-pantry/test-1/acc530466eb811eeb8b60...</td>\n",
+       "      <td>LocalFileSystem</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                               uuid                      upload_time  \\\n",
+       "0  acc530466eb811eeb8b60242ac1f000a 2023-10-19 19:49:54.672040+00:00   \n",
+       "\n",
+       "  parent_uuids basket_type label weave_version  \\\n",
+       "0           []      test-1               1.1.0   \n",
+       "\n",
+       "                                             address     storage_type  \n",
+       "0  weave-demo-pantry/test-1/acc530466eb811eeb8b60...  LocalFileSystem  "
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pantry1.upload_basket(upload_items=[{'path':'WeaveDemoText.txt', 'stub':False}], basket_type=\"test-1\", metadata = {'Data Type':'text'})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e3f152e7-5735-451a-8864-7e8087c66305",
+   "metadata": {},
+   "source": [
+    "To easily manipulate our baskets we can convert our pantry into a dataframe. "
```

Comment:
we export the index as a dataframe. "Converting" a pantry would imply moving or manipulating the data inside a pantry somehow.